config-example.yml

```diff
@@ -0,0 +1,140 @@
+graphhopper:
+
+  # OpenStreetMap input file
+  datareader.file: core/files/monaco.osm.gz
+
+  ##### Vehicles #####
+
+
+  # Possible options: car,foot,bike,bike2,mtb,racingbike,motorcycle (comma separated)
+  # bike2 takes elevation data into account (like up-hill is slower than down-hill) and requires enabling graph.elevation.provider below
+  graph.flag_encoders: car
+
+
+  # Enable turn restrictions for car or motorcycle.
+  # graph.flag_encoders: car|turn_costs=true
```

Comment:
In Yaml, we might run into issues when using special characters like | or =. Did you try if this works or if we need to wrap the whole string ins quotes?